Draft for quiz submission

davidcramb opened the pull request.

davidcramb committed:
argh

davidcramb committed:
Refactoring of code & major bug fix
Refactoring code for IIFE's to augment the event handler IIFE. Created new js file called eventaug to hold hte functions that the event handlers will run. Cleaned up functions to make them more single function.
Also, fixed a major bug where the click event to modify card styling bubbled to the wrong elements.

gregkorte commented:
Hey David. Looks like you still have a bit to go. First, if I click on a description I can change it. If I click on another one still only the first one changes. The borders have to be clicked twice to revert to normal. I have some questions for you about the way you structured your `Carlot.js` & `quiz.js`. We can chat about that Monday. As for the DOM traversal, I would be cautious using things like `.children[2]` and `offsetParent` in case you ever want to change the layout of your page. Instead I would use an id to get the elements I wanted to use. Last be careful about leaving trailing whitespace in your files. It can and will eventually cause merge conflicts, and it's better to just avoid that. When we talk about your files Monday I can show you a tool that will identify trailing whitespace in your files.

davidcramb committed:
Bug Fix and Clean Up
Added function to prevent multiple cards from containing the clickedCard class. Removed trailing whitespace from code. Submit for PR.

davidcramb commented:
Sorry for the delay! I opted not to change how the event handlers are structured (I have a good handle on how to work them and sometimes it's valuable to go down the wrong rabbit hole) so I've left those in place and fixed having multiple instances of edit-able cards.

gregkorte commented:
Looks good, David. Sorry this feedback took so long to get back to you. You are good to merge. :+1:
